packages/app/src/debug/DebugLoading.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    data-cy="debug-loading"
+    class="p-30px"
+  >
+    <div>
+      <div class="flex items-center">
+        <div class="rounded-full bg-gray-50 h-16px w-344px" />
+        <DebugLoadingDivider />
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-120px" />
+      </div>
+
+      <div class="flex mt-15px items-center">
+        <div class="rounded-full bg-gray-50 h-16px w-72px" />
+        <div class="rounded-full bg-gray-50 h-16px ml-8px w-160px" />
+        <div class="rounded-full bg-gray-50 h-16px ml-8px w-72px" />
+        <DebugLoadingDivider />
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-80px" />
```

Comment:
```suggestion
	<template v-for="n in 4">
          <DebugLoadingDivider />
          <div class="rounded-full bg-gray-50 h-8px ml-8px w-80px" />
        </template>
```
Nit-pick:  you can repeat this same content 4 times using this `v-for` syntax https://vuejs.org/guide/essentials/list.html#v-for-with-a-range

There are other places in this file you could do that with as well.